src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -71,6 +76,22 @@ def run(self, opname, input_data, wkaoi=None, cache_key=''):
     data = settings.GEOP['json'][opname].copy()
     data['input'].update(input_data)
 
+    # Populate layers
+    layer_config = dict(settings.GEOP['layers'], **layer_overrides)
+
+    try:
+        if 'targetRaster' in data['input']:
+            data['input']['targetRaster'] = use_layer(
+                data['input']['targetRaster'], layer_config)
+
+        for idx, raster in enumerate(data['input']['rasters']):
+            data['input']['rasters'][idx] = use_layer(
+                raster, layer_config)
+    except Exception as x:
+        return {
+            'error': str(x)
+        }
```

Comment:
That's a good suggestion. We pass the errors on because of Celery reasons, but recording the stacktrace explicitly would be useful. Made #3408 for it.